paddle/framework/data_transform.cc

```diff
@@ -111,5 +111,18 @@ void TransDataLayout(const platform::DeviceContext* ctx,
 }  // namespace paddle
 
 namespace f = paddle::framework;
+
+std::vector<int> NHWC2NCHW = {0, 3, 1, 2};
+std::vector<int> NCHW2NHWC = {0, 2, 3, 1};
```

Comment:
Is it more appropriate to put this in an anonymous namespace?

Reply:
.cc file always safe with translate unit local object. But you are right, anonymous namespace or static keywords is better. 
Done.